common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -875,6 +875,23 @@ def user_details_force_sync(auth_entry, strategy, details, user=None, *args, **k
                 changed[provider_field] = current_value
                 setattr(model, field, provider_value)
 
+        # Generate fullname only for IES IDP.
+        try:
+            if (not details.get('fullname') and details.get('first_name') and details.get('last_name') and
+                any(['iam-stage.pearson.com' in current_provider.entity_id, 'iam.pearson.com' in current_provider.entity_id])):  # pylint: disable=line-too-long
+                fullname_value = '%s %s' % (details.get('first_name'), details.get('last_name'))
+                changed['fullname'] = fullname_value
+
+                setattr(user.profile, 'name', fullname_value)
+        except Exception as error:  # pylint: disable=broad-except
```

Comment:
From a personal point of view, I don't like broad exceptions, what could possibly go wrong with this code enough to throw an exception? @ivanvgh

Reply:
Actually there's nothing that can throw an exception, @diegomillan I am removing this, if there is something you notice to preserve this 'try except' let me know please.